corehq/apps/locations/dbaccessors.py

```diff
@@ -158,3 +158,17 @@ def get_location_ids_with_location_type(domain, location_type_code):
         is_archived=False,
         location_type__code=location_type_code,
     ).values_list('location_id', flat=True)
+
+
+def get_practice_mode_mobile_workers(domain):
```

Comment:
a cached version of this would also be good since the response isn't going to change often